SingularityService/src/main/java/com/hubspot/singularity/smtp/MailTemplateHelpers.java

```diff
@@ -46,15 +47,32 @@ public MailTemplateHelpers(SandboxManager sandboxManager, SingularityConfigurati
     this.smtpConfiguration = singularityConfiguration.getSmtpConfiguration();
   }
 
+  public List<SingularityMailTaskMetadata> getJadeTaskMetadata(Collection<SingularityTaskMetadata> taskMetadata) {
+    List<SingularityMailTaskMetadata> output = Lists.newArrayListWithCapacity(taskMetadata.size());
+
+    for (SingularityTaskMetadata metadataElement : taskMetadata) {
+      output.add(
+              new SingularityMailTaskMetadata(
+                      DateFormatUtils.formatUTC(metadataElement.getTimestamp(), TASK_DATE_PATTERN),
+                      metadataElement.getType(),
+                      metadataElement.getTitle(),
+                      metadataElement.getUser().or(""),
+                      metadataElement.getMessage().or(""),
+                      metadataElement.getLevel().toString()));
+    }
+
+    return output;
+  }
+
   public List<SingularityMailTaskHistoryUpdate> getJadeTaskHistory(Collection<SingularityTaskHistoryUpdate> taskHistory) {
     List<SingularityMailTaskHistoryUpdate> output = Lists.newArrayListWithCapacity(taskHistory.size());
 
     for (SingularityTaskHistoryUpdate taskUpdate : taskHistory) {
       output.add(
-          new SingularityMailTaskHistoryUpdate(
-              DateFormatUtils.formatUTC(taskUpdate.getTimestamp(), TASK_DATE_PATTERN),
-              WordUtils.capitalize(taskUpdate.getTaskState().getDisplayName()),
-              taskUpdate.getStatusMessage().or("")));
+              new SingularityMailTaskHistoryUpdate(
+                      DateFormatUtils.formatUTC(taskUpdate.getTimestamp(), TASK_DATE_PATTERN),
```

Comment:
might want to double check your editor settings for indentations. Not a huge deal, but easier to look at the correct diffs if there aren't also indentation changes